code/part-two/processor/tests/09-ExtraCredit-AcceptResponse.js

```diff
@@ -166,17 +163,11 @@ describe('Accept Response', function() {
   it('should reject acceptances for indexes with no response', function() {
     const payload = { action: 'ACCEPT_RESPONSE', offer, response: 1 };
     const txn = new Txn(payload, ownerPriv);
+    const submission = handler.apply(txn, context);
 
-    return handler.apply(txn, context)
-      .catch(err => {
-        expect(err, 'Error should be an InvalidTransaction')
-          .to.be.instanceOf(InvalidTransaction);
-        return true;
-      })
+    return expect(submission).to.be.rejectedWith(InvalidTransaction)
       .then(wasRejected => {
```

Comment:
I think `wasRejected` is no longer being used any more

Reply:
Commits could be out of order in the PR (GitHub does weird orders with rebases sometimes). As of this commit, `wasRejected` is still used. Later this whole messy promise construct gets refactored into a custom chai method: `rejectedWith`.